tests/test_navbar.py

```diff
@@ -0,0 +1,447 @@
+
+    def test_sign_in_button(self):
+        assert len(self.base_page.navbar.sign_in_button) == 1
+        self.base_page.navbar.sign_in_button.click()
+        assert 'login' in self.driver.current_url
+
+    def test_sign_in_button_not_present_if_login_in(self):
+        login(self.base_page)
+        assert len(self.base_page.navbar.sign_in_button) == 0
+
+    def test_sign_up_button(self):
+        assert len(self.base_page.navbar.sign_up_button) == 1
+        self.base_page.navbar.sign_in_button.click()
+        assert 'register' in self.driver.current_url
+
+    def test_sign_up_button_not_present_if_login_in(self):
+        login(self.base_page)
+        assert len(self.base_page.navbar.sign_up_button) == 0
+
+    @classmethod
+    def teardown_class(cls):
+        cls.driver.quit()
+
+
+class TestPreprintsPageNavBar:
+
+    @classmethod
+    def setup_class(cls):
+        cls.driver = launch_driver()
+
+    def setup_method(self, method):
+        self.preprint_page = PreprintPage(self.driver)
+        self.preprint_page.goto()
+
+    def test_osf_home_drop_down_home(self):
+        self.preprint_page.navbar.home.click()
+        assert self.driver.current_url is settings.OSF_HOME
+
+    def test_osf_home_drop_down_preprints(self):
+        self.preprint_page.navbar.preprints.click()
+        preprints_url = settings.OSF_HOME + '/preprints/'
+        assert self.driver.current_url is preprints_url
+
+    def test_osf_home_drop_down_registries(self):
+        self.preprint_page.navbar.registries.click()
+        registries_url = settings.OSF_HOME + '/registries/'
+        assert self.driver.current_url is registries_url
+
+    def test_osf_home_drop_down_meetings(self):
+        self.preprint_page.navbar.meetings.click()
+        meetings_url = settings.OSF_HOME + '/meetings/'
+        assert self.driver.current_url is meetings_url
+
+    def test_add_a_preprint_not_login(self):
+        self.preprint_page.navbar.add_a_preprint.click()
+        assert 'login' in self.driver.current_url
+
+    def test_add_a_preprint_login(self):
+        self.preprint_page.navbar.add_a_preprint.click()
+        add_preprint_url = settings.OSF_HOME + '/preprints/submit/'
+        assert self.driver.current_url is add_preprint_url
+
+    def test_nagivation_bar_link_search(self):
+        self.preprint_page.search.click()
+        search_url = settings.OSF_HOME + '/search/'
+        assert self.driver.current_url is search_url
+
+    def test_nagivation_bar_link_support(self):
+        self.preprint_page.support.click()
+        support_url = settings.OSF_HOME + '/support/'
+        assert self.driver.current_url is support_url
+
+    def test_nagivation_bar_link_donate(self):
+        self.preprint_page.donate.click()
+        assert 'cos.io/donate-to-cos' in self.driver.current_url
+
+    def test_user_profile_menu_profile_not_there_if_not_login(self):
+        assert len(self.preprint_page.navbar.user_dropdown_profile) == 0
```

Comment:
If the element is expected to not be there, it will always raise an exception. I believe you should use `with pytest.raises(ValueError):`